.circleci/config.yml

```diff
@@ -74,11 +74,7 @@ jobs:
 
       - run:
           name: Publish package LIB
-          command: npm publish lib
-
-      - run:
-          name: Publish package ES
-          command: npm publish es
+          command: npm publish dist
```

Comment:
this is just `npm publish`!

Reply:
Thanks. That makes sense 🙂 

Somehow I forgot to merge pull `master` first and looking at the `package.json` it was requiring the index 🤦 My brain was probably very slow yesterday.